cassandra-analytics-release/prepare_release.sh

```diff
@@ -0,0 +1,349 @@
+    exit 1
+fi
+
+if curl --output /dev/null --silent --head --fail "https://github.com/apache/cassandra-analytics/tree/${release}-tentative" ; then
+    echo "The release candidate tag for ${release}-tentative is already at https://github.com/apache/cassandra-analytics/tree/${release}-tentative"
+    exit 1
+fi
+
+if curl --output /dev/null --silent --head --fail "https://github.com/apache/cassandra-analytics/tree/cassandra-analytics-${release}" ; then
+    echo "The published release tag for ${release} is already at https://github.com/apache/cassandra-analytics/tree/cassandra-analytics-${release}"
+    exit 1
+fi
+
+if git tag -l | grep -q "${release}-tentative"; then
+    echo "Local git tag for ${release}-tentative already exists"
+    exit 1
+fi
+
+head_commit=`git log --pretty=oneline -1 | cut -d " " -f 1`
+
+echo "Preparing release for $release from commit:"
+
+echo ""
+git show $head_commit
+java -version
+
+echo "Is this what you want?"
+select yn in "Yes" "No"; do
+    case $yn in
+        Yes) break;;
+        No) echo "Alright, come back when you've made up your mind"; exit 0;;
+    esac
+done
+
+# "Saves" stdout to other descriptor since we might redirect them below
+exec 3>&1 4>&2
+
+if [ $verbose -eq 0 ]
+then
+    # Not verbose, redirect all output to a logfile
+    logfile="vote-${release}.log"
+    [ ! -e "$logfile" ] || rm $logfile
+    touch $logfile
+    exec > $logfile
+    exec 2> $logfile
+fi
+
+execute()
+{
+    local cmd=$1
+
+    echo ">> $cmd"
+    [ $fake_mode -eq 1 ] || $cmd
+    if [ $? -ne 0 ]
+    then
+        echo "Error running $cmd" 1>&3 2>&4
+        exit $?
+    fi
+}
+
+current_dir=`pwd`
+tmp_dir=`mktemp -d`
+distributions_dir=${tmp_dir}/cassandra-analytics/build/distributions
+
+echo "Tagging release ..." 1>&3 2>&4
+execute "git tag $release-tentative"
+execute "git push $git_asf_remote refs/tags/$release-tentative"
+
+echo "Cloning fresh repository ..." 1>&3 2>&4
+execute "cd $tmp_dir"
+## We clone from the original repository to make extra sure we're not screwing, even if that's definitively slower
+execute "git clone $asf_git_repo/cassandra-analytics.git"
+
+echo "Building and uploading artifacts ..." 1>&3 2>&4
+execute "cd $tmp_dir/cassandra-analytics"
+execute "git checkout -b $release-tentative"
+# Build java 11 artifacts and publish
+execute "./scripts/build-dependencies.sh"
+execute "./gradlew --no-daemon -Dorg.gradle.java.home=${java_11_home} clean"
+execute "./gradlew --no-daemon -Pscala=2.12 -P-Dorg.gradle.java.home=${java_11_home} -PforceSigning -Prelease=true -Pversion=${release} assemble publish --stacktrace"
+execute "./gradlew --no-daemon -Pscala=2.13 -Dorg.gradle.java.home=${java_11_home} -PforceSigning -Prelease=true -Pversion=${release} assemble publish --stacktrace"
+
+echo "Artifacts uploaded, find the staging repository on repository.apache.org, \"Close\" it, and indicate its staging number:" 1>&3 2>&4
+read -p "staging number Java 11? " staging_number 1>&3 2>&4
```

Comment:
I think we should collect values for both builds :

- scala 2.12, java11, spark 3
- scala 2.13, java11, spark 3

Both staging numbers should be included in the email text below

Reply:
no, I believe they will be 2 different staging numbers. In Sidecar, we get two staging numbers , one for java 8 artifacts, and one for java 11 artifacts.